docs/manuals/index.md

```diff
@@ -275,6 +275,40 @@ As of today, the query caching stores:
   for a specific entityType are removed only if a entityType is dropped, not
   in case all its values are removed.
 
+**Work Queue Cache**
+QL uses the work queue for NGSI notifications. When an entity payload
+comes through the notify endpoint, the Web app turns the payload
+into a task to save it to the DB, adds the task to the queue and returns
+a `200` to the client immediately. A separate instance of QL, configured as
+a queue worker, fetches the task from the queue and runs it to actually insert
+the NGSI entities into the DB, possibly retrying the insert at a later time
+if it fails. Clients connect to the Web app to manage notify tasks
+in the queue.
+When using a work queue, there are two sets of QuantumLeap processes-
+set of Web servers that add tasks to a queue and a pool of queue worker
+processes that fetch tasks from the queue and run them asynchronously.
+The Web servers are QuantumLeap Web app instances configured to offload
+tasks to the work queue through the various `WQ_*` environment variables
+available for the Web app. A worker processes is a Python interpreter loaded
+with the same code as the Web app but started with a different entry point.
+To configure you can use the following environment variables-
+
+- `WQ_OFFLOAD_WORK` - `Whether to offload insert tasks to a work queue
+default: False`
+- `WQ_RECOVER_FROM_ENQUEUEING_FAILURE` - `Whether to run tasks immediately
+if a work queue is not available. Default: False`
+- `WQ_MAX_RETRIES` - `How many times work queue processors should retry failed
+tasks. Default: 0 (no retries).`
+- `WQ_FAILURE_TTL` - `How long, in seconds, before removing failed tasks from
+the work queue. Default: 604800 (a week).`
+- `WQ_SUCCESS_TTL` - `How long, in seconds, before removing successfully run
+tasks from the work queue.Default: 86400 (a day)`
+- `WQ_WORKERS` - `How many worker queue processors to spawn`
+
```

Comment:
should we add a link to the config section where these vars are documented fully? e.g. further info about those vars in the [link to config].

Reply:
@c0c0n3 , Sure i will add link and update commit.